book/actix-plugin.md

```diff
@@ -8,10 +8,10 @@ Let's start with a simple actix-web application. It has `actix-web` and `serde`
 # [package] ignored for brevity
 
 [dependencies]
-actix-web = "2.0"
+actix-web = "3.0"
```

Comment:
Can you leave a comment here mentioning that 2.0 is also supported?

Reply:
done, can you check plz

Reply:
We can just leave it at, `actix-web 2.0 is also supported`

Reply:
OK, done